metadata/metadata_service/api/feature.py

```diff
@@ -0,0 +1,131 @@
+from typing import Any, Iterable, Mapping, Union
+
+from flasgger import swag_from
+from flask_restful import Resource, reqparse
+
+from metadata_service.api.badge import BadgeCommon
+from metadata_service.api.tag import TagCommon
+from metadata_service.proxy import get_proxy_client
+
+
+class FeatureDetailAPI(Resource):
+    """
+    FeatureDetail API
+    """
+
+    def __init__(self) -> None:
+        self.client = get_proxy_client()
+
+    @swag_from('swagger_doc/feature/detail_get.yml')
+    def get(self, feature_uri: str) -> Iterable[Union[Mapping, int, None]]:
+        pass
+
+
+class FeatureLineageAPI(Resource):
+
+    def __init__(self) -> None:
+        self.client = get_proxy_client()
+
+    @swag_from('swagger_doc/table/lineage_get.yml')
+    def get(self, feature_uri: str) -> Iterable[Union[Mapping, int, None]]:
+        pass
+
+
+class FeatureStatsAPI(Resource):
+
+    # TODO integrate stats source for FE
+    def __init__(self) -> None:
+        self.client = get_proxy_client()
+
+    @swag_from('swagger_doc/feature/detail_get.yml')
+    def get(self, feature_uri: str) -> Iterable[Union[Mapping, int, None]]:
+        pass
+
+
+class FeatureGenerationCodeAPI(Resource):
+
+    # TODO use Query common model
+    def __init__(self) -> None:
+        self.client = get_proxy_client()
+
+    @swag_from('swagger_doc/feature/detail_get.yml')
+    def get(self, feature_uri: str) -> Iterable[Union[Mapping, int, None]]:
+        pass
+
+
+class FeatureSampleAPI(Resource):
+
+    # TODO use DataSample common model
+    def __init__(self) -> None:
+        self.client = get_proxy_client()
+
+    @swag_from('swagger_doc/feature/detail_get.yml')
```

Comment:
FeatureStatsAPI, FeatureGenerationCodeAPI, FeatureSampleAPI have the same file in `@swag_from` - is it correct?